template/Dockerfile.oracle

```
@@ -130,6 +130,9 @@ RUN set -eux; \
 VOLUME /var/lib/mysql
 
 COPY docker-entrypoint.sh /usr/local/bin/
+{{ if [ "5.7", "8.0" ] | index(env.version) then ( -}}
+RUN ln -s usr/local/bin/docker-entrypoint.sh /entrypoint.sh # backwards compat
```

Should this use an absolute path? (just to be sure / explicit)?
```suggestion
RUN ln -s /usr/local/bin/docker-entrypoint.sh /entrypoint.sh # backwards compat
```

---

(I guess the image would unlikely get a WORKDIR changed/set, but if it would..)

---

:shrug: it's put in the root, and this matches the Debian version :innocent:

---

(Semi-intentional that it's not "pretty" so that we definitely get rid of it later :eyes:)

---

🤷 alright, you win 😂💜

---

No hard(link) feelings 🤣